astropy/io/votable/tests/table_test.py

```diff
@@ -224,6 +224,20 @@ def test_binary2_masked_strings():
     astropy_table.write(output, format='votable')
 
 
+def test_validate_output_type():
+    """
+    Issue #12603
+    """
+    votable_filepath = get_pkg_data_filename('data/regression.xml')
+    # When output is None, check that validate returns validation output as a string
+    validate_out = validate(votable_filepath, output=None)
+    assert type(validate_out) == str
+
+    # When output is not set, check that validate returns a bool
+    validate_out = validate(votable_filepath)
+    assert type(validate_out) == bool
```

Comment:
Any advantage to get for `type` over `isinstance`?

```suggestion
    assert isinstance(validate_out, bool)
```

Reply:
Fixed this in recent commits.